src/network-workloads/model-net-synthetic-dfly-plus.c

```diff
@@ -308,6 +656,22 @@ static void handle_remote_event(
         (void)m;
         (void)lp;
 	ns->msg_recvd_count++;
+
+    if(m->dest != ns->svr_id)
+        tw_error(TW_LOC, "Error: msg issued by %llu at time %lfns should arrive at %llu but reached %llu\n", (unsigned long long)m->src, m->issued_time , (unsigned long long) m->dest, (unsigned long long) ns->svr_id);
+
+    m->arrive_time = tw_now(lp);
+    ns->total_comm_time += (m->arrive_time - m->issued_time);
```

Comment:
In the beginning, I used this variable to count the sum of latency of every message, then divided by the total number of messages to give the average. It becomes unnecessary to keep this since later I added an IO file to store each message's latency. 
I deleted this and will show the modification in the next commit.